edk2basetools/VfrCompiler/IfrUtility.py

```diff
@@ -0,0 +1,2238 @@
+import ctypes
```

Comment:
Should use `is` or `is not` when comparing variables with `None`, `True`, `False` throughout this file.